makejdk-any-platform.sh

```diff
@@ -24,31 +24,95 @@
 
 # You can set the JDK boot directory with the JDK_BOOT_DIR environment variable
 
-set -eu
-
-OS_MACHINE_NAME=$(uname -m)
+export OS_KERNEL_NAME=""
 OS_KERNEL_NAME=$(uname | awk '{print tolower($0)}')
+export OS_MACHINE_NAME=""
+OS_MACHINE_NAME=$(uname -m)
+
+export OPENJDK_VERSION=""
+export REPOSITORY=""
 
-if [[ "$OS_MACHINE_NAME" == "s390x" ]] ; then
-   export JVM_VARIANT=${JVM_VARIANT:-zero}
-   export BUILD_FULL_NAME=${BUILD_FULL_NAME:-linux-s390x-normal-${JVM_VARIANT}-release}
-   S390X_MAKE_ARGS="CONF=${BUILD_FULL_NAME} DEBUG_BINARIES=true images"
-   export MAKE_ARGS_FOR_ANY_PLATFORM=${MAKE_ARGS_FOR_ANY_PLATFORM:-$S390X_MAKE_ARGS}
+counter=0
+for i in "$@"; do
+  let counter++
+  case "$i" in
+    "--version" | "-v")
+      let counter++
+      string="\$$counter"
+      OPENJDK_VERSION=$(echo "$@" | awk "{print $string}")
+      ;;
+  esac
+done
+
+if [ "$OPENJDK_VERSION" == "jdk9" ]; then
+  export JDK_PATH="jdk"
+elif [ "$OPENJDK_VERSION" == "jdk8u" ]; then
+  export JDK_PATH="j2sdk-image"
+else
+  echo "Please specify a version with --version or -v , either jdk9 or jdk8u"
+  man ./makejdk-any-platform.1
+  exit 1
 fi
-if [[ "$OS_MACHINE_NAME" = "ppc64le" ]] ; then
+
+REPOSITORY="${REPOSITORY:-adoptopenjdk/openjdk-$OPENJDK_VERSION}";
+REPOSITORY="$(echo "${REPOSITORY}" | awk '{print tolower($0)}')";
+
+case "$OS_MACHINE_NAME" in
+"s390x")
+  case "OPENJDK_VERSION" in
+     "jdk9")
+     export JVM_VARIANT=${JVM_VARIANT:-server}
+     export CONFIGURE_ARGS_FOR_ANY_PLATFORM=${CONFIGURE_ARGS_FOR_ANY_PLATFORM:-"--disable-warnings-as-errors"};;
```

Comment:
Since this line is repeating across all platforms, can we not use a separate case statement and set the  `CONFIGURE_ARGS_FOR_ANY_PLATFORM` for `jdk9` and not for `jdk8u`